src/Sarif.Multitool.FunctionalTests/TestData/UriBaseIdRequiresRelativeUri/AbsoluteUriInResultGraphNode_Expected.sarif

```diff
@@ -21,12 +21,12 @@
           "ruleId": "SARIF1014",
           "level": "error",
           "message": {
+            "messageId": "Default",
             "arguments": [
-              "/runs/0/results/0/graphs/0/nodes/0/location/physicalLocation/fileLocation/uri",
+              "/runs/0/results/0/graphs/g1/nodes/0/location/physicalLocation/fileLocation/uri",
```

Comment:
>g1 [](start = 40, length = 2)

As a result, the JSON pointers for elements of the `graphs` property have changed.